enterprise/server/remote_execution/filecache/filecache.go

```diff
@@ -288,6 +288,29 @@ func (c *fileCache) FastLinkFile(ctx context.Context, node *repb.FileNode, outpu
 	return true
 }
 
+func (c *fileCache) Open(ctx context.Context, node *repb.FileNode) (f *os.File, err error) {
```

Comment:
We've found the named return params to be a little error-prone in the past.

Reply:
It's consistent with the other filecache functions so I'm going to keep this one as well. If we change this we should do this for everything in this file.